extensions/wikia/CategorySelect/js/CategorySelect.view.js

```diff
@@ -77,7 +77,8 @@
 							controller: 'CategorySelectController',
 							data: {
 								articleId: articleId,
-								categories: $wrapper.data( 'categorySelect' ).getData( '.new' )
+								categories: $wrapper.data( 'categorySelect' ).getData( '.new' ),
+								token: mw.user.tokens.get('editToken')
```

Comment:
Not to open a can of worms, but I don't see where  the thrown exceptions (the existing one and one added in this PR) are caught - what will be the end state?